src/cast_core/src/CollectionScanner.cpp

```diff
@@ -516,7 +516,7 @@ void CollectionScanner::run() {
 CollectionScanner::CollectionScanner(genny::ActorContext& context)
     : Actor{context},
       _totalInserts{context.operation("Insert", CollectionScanner::id())},
-      _client{context.client()},
+      _client{context.client(v1::DEFAULT_CLIENT_NAME, CollectionScanner::id() )},
```

Comment:
At the risk of introducing chaos, should we consider switching the order of the arguments? Then we could use the default value for the client name in all these files.

Reply:
I did consider that but decided to put it the change out for review first. It might make the scale of the changes marginally smaller.

There is also another option to consider, should the configuration be on the Client or the Actor? The scale of changes for an Actor configuration would be about the same (and I don't think there would be any changes required to PoolManager ). However, I *think* the configuration being on the client pool itself makes more sense and is less verbose in the yml files.

Reply:
Alright, I'm good. I'll leave this thread to sit for a bit in case Ryan has anything to add.